devito/symbolics/inspection.py

```diff
@@ -1,10 +1,9 @@
 from collections import Counter
-
```

Comment:
this should remain -- imports from stdlib separated from imports from our own deps